### PR TITLE
Feature/new frontend

### PR DESCRIPTION
**Pull request by mfeckie**
@hahuang65 This swaps out to a new frontend, which consumes the API we've worked on already.

(There's some API stuff in this PR because of another PR which awaits review)

For full info, the frontend is here https://github.com/mfeckie/beaker-frontend .  There are screenshots over there too.  I tool the liberty of making a logo for beaker, but you may not like it, so can remove as you see fit.

Anyway, take a look and let me know your thoughts

### CODE REVIEW

File: priv/static/js/beaker-frontend.js [hunk 1]
```diff
@@ -0,0 +1 @@
...MTguMDM2bDAsLTM2LjA3M2MwLC05Ljk1NCA4LjA4MSwtMTguMDM1IDE4LjAzNiwtMTguMDM2bDM0OS44ODYsMFoiIHN0eWxlPSJmaWxsOm5vbmU7c3Ryb2tlLXdpZHRoOjMycHg7c3Ryb2tlOiMwMDA7Ii8+PHBhdGggZD0iTTI3OC4zNTQsOTIuNDgzbDE5My4wOTksMC4wMjYiIHN0eWxlPSJmaWxsOm5vbmU7c3Ryb2tlLXdpZHRoOjE3cHg7c3Ryb2tlOiMwMDA7Ii8+PC9nPjwvZz48L2c+PHRleHQgeD0iLTE2LjMzNnB4IiB5PSI3MzUuMTAzcHgiIHN0eWxlPSJmb250LWZhbWlseTonT3BlblNhbnMtRXh0cmFib2xkJztmb250LXNpemU6MjA2LjE4N3B4O2ZpbGw6I2ZmZjtzdHJva2Utd2lkdGg6MXB4O3N0cm9rZTojMDAwOyI+Qjx0c3BhbiB4PSIxMjguNDUxcHggMjYxLjI3N3B4IDM5NS4xNTJweCA1MzIuNDlweCA2NjUuMzE2cHggIiB5PSI3MzUuMTAzcHggNzM1LjEwM3B4IDczNS4xMDNweCA3MzUuMTAzcHggNzM1LjEwM3B4ICI+ZWFrZXI8L3RzcGFuPjwvdGV4dD48L2c+PC9zdmc+"),e.appendChild(t,n),t},buildRenderNodes:function(){return[]},statements:[],locals:[],templates:[]}}())}),define("beaker-frontend/config/environment",["ember"],function(e){var t="beaker-frontend";try{var n=t+"/config/environment",a=e["default"].$('meta[name="'+n+'"]').attr("content"),r=JSON.parse(unescape(a));return{"default":r}}catch(i){throw new Error('Could not read config from meta tag with name "'+n+'".')}}),runningTests||require("beaker-frontend/app")["default"].create({name:"beaker-frontend",version:"0.0.0+f9f2cc14"});
```
Comment (mfeckie) [inline priv/static/js/beaker-frontend.js:1]: OH @houndci doesn't like the minified JS :cry:

Comment (hahuang65) [inline priv/static/js/beaker-frontend.js:1]: hahaha, wonder what we should do about that, and if it'd keep warning everytime we do something...

**Comment by hahuang65** [comment]
@mfeckie this looks cool! Got a question: how do the two apps link together? Would they have to install `beaker-frontend` separately?

**Comment by mfeckie** [comment]
@hahuang65 The concept would be that the frontend is available to develop separately but the assets get compiled and copied into beaker.  This would ensure that users don't have to take any action to use it, but keeps them as separate concerns.

File: priv/static/js/beaker-frontend.js (hunk 1 above)
Comment (mfeckie) [inline priv/static/js/beaker-frontend.js:1]: There's probably a flag we can pass to tell it to ignore for the particular file

**Pull request merged**